python/lsst/daf/butler/datastore/stored_file_info.py

```diff
@@ -239,14 +280,15 @@ def from_record(cls: type[StoredFileInfo], record: Mapping[str, Any]) -> StoredF
         storageClass = cls.storageClassFactory.getStorageClass(record["storage_class"])
         component = record["component"] if (record["component"] and record["component"] != NULLSTR) else None
 
+        # UUID may be converted to string, e.g. in round-trip through JSON,
+        # convert it back to UUID.
```

Comment:
This comment doesn't seem to be describing any code around it.

Reply:
Leftover for previous commits, I dropped dataset_id from the class, will remove this comment.